packages/core-v2-sdk/src/models/Pool.ts

```diff
@@ -30,6 +32,25 @@ export class Pool {
     this.maturityDate = new Date(id * 1000);
   }
 
+  /**
+   * @async
+   * Gets the associated MultiTerm model for this pool.
+   * @return {Promise<YieldSource | null>}
+   */
+  getMultiPool(): Promise<MultiTerm> {
```

Comment:
wait this is confusing, why does `getMultiPool` return a multiTerm?

Reply:
Woops! My bad, good catch